host/bcm283x/dma.go

```diff
@@ -514,11 +515,7 @@ func (d *dmaChannel) isAvailable() bool {
 // It doesn't clear the local controlBlock cached values.
 func (d *dmaChannel) reset() {
 	d.cs = dmaReset
-	// Clear bits if needed.
```

Comment:
You should rebase on origin/master. In short:

git rebase -i origin/master
# squash them all
git mergetool
git push -f origin dma1pull

You'll want to configure vimdiff or something similar. e.g.:
https://github.com/maruel/bin_pub/blob/master/configs/.gitconfig#L71
There's a squash alias in there if you want.

You can also use a merge flow but since I always squash, it's easier to rebase locally. That keeps the history purely linear, which I prefer.

Reply:
I guess I correctly updated my origin/master and merged them this time.